modules/cbMap/cbMap.js

```diff
@@ -6,7 +6,60 @@
  * Portions created by vtiger are Copyright (C) vtiger.
  * All Rights Reserved.
  ************************************************************************************/
+var sweetAlert = document.createElement('script');  
+sweetAlert.setAttribute('src','https://unpkg.com/sweetalert/dist/sweetalert.min.js');
```

Comment:
We are following [lightning design system](https://www.lightningdesignsystem.com/) recommendations. I would like to avoid introducing another external library and ask you to use [lightning design system Popovers](https://www.lightningdesignsystem.com/guidelines/messaging/components/popovers/) if possible.